api/configuration/references.ts

```diff
@@ -50,6 +50,9 @@ export const FEATURE_TERMS_AND_CONDITIONS_ENABLED = 'termsAndConditionsEnabled'
 export const FEATURE_HELMET_ENABLED = 'helmetEnabled'
 export const FEATURE_REDIS_ENABLED = 'redisEnabled'
 
+export const LINKS_MANAGE_CASES_LINK = 'links.manageCaseLink'
```

Comment:
We dont need Manage Cases for MO. Can we remove it?

Reply:
No We need both link in the MO register page